web-local/src/routes/(application)/dashboard/[name]/+page.ts

```diff
@@ -10,28 +12,26 @@ export async function load({ params }) {
   const localConfig = await runtimeServiceGetConfig();
 
   try {
-    const dashboardMeta = await runtimeServiceGetCatalogEntry(
+    await runtimeServiceGetFile(
       localConfig.instance_id,
-      params.name
+      `dashboards/${params.name}.yaml`
     );
-
-    const dashboardYAML = dashboardMeta?.entry?.metricsView;
-
-    // check if metrics definition is defined
-    if (dashboardYAML) {
-      return {
-        metricViewName: params.name,
-      };
-    }
   } catch (err) {
-    if (
-      ExplorerMetricsDefinitionDoesntExist.includes(err.message) ||
-      err.message.includes(ExplorerMetricsDefinitionDoesntExist)
-    ) {
+    if (err.response?.data?.message.includes("entry not found")) {
```

Comment:
do we get anything like an enum for this? Feels a little weird that we have these hard-coded.

Reply:
Good callout – no enum from the backend, but I created a `lib/errors/messages.ts` to quarantine the hard-coded text